balancer-labs/frontend-v2
# Fixing the coin image layout jank in the veBAL header

# Description

Some of the coins in the veBAL page hero display too large which causes layout jank. This PR fixes this issue.

## Type of change


## How should this be tested?

Load the /vebal page and watch the hero. Ideally, the coin images should load gracefully.
